runtime/doc/options.txt

```diff
@@ -6120,6 +6120,31 @@ A jump table for the options with a short description can be found at |Q_op|.
 	NOTE: This option is set to the Vi default value when 'compatible' is
 	set and to the Vim default value when 'compatible' is reset.
 
+
+						*'numquickfix'* *'numqf'*
+'numquickfix' 'numqf'	number	(default: 10)
```

Comment:
The `chistory` command is used to display the quickfix stack and the `lhistory` command is used to display the location list stack.  So it make sense to call these options `chistory` and `lhistory`.  This also is similar to the existing `history` option.